composer/distributed/dist_strategy.py

```diff
@@ -213,6 +219,7 @@ def prepare_fsdp_module(
         precision: (Precision): The precision being used by the Trainer, used to fill in defaults for FSDP `mixed_precision` settings.
         device (Device): The device being used by the Trainer.
         auto_microbatching (bool, optional): Whether or not auto microbatching is enabled.
+        using_tp (bool, optional): Whether the model has been wrapped with Tensor Parallelism, in which case only a single optimizer param group is supported.
```

Comment:
new arg, perhaps there's an easier way to determine if using tp, but this was what made sense to me. would also appreciate feedback regarding this.

Reply:
can we just check the param groups in `prepare_tp_module`